apps/site/src/app/(main)/(home)/sections/FAQ/FAQAccordionDesktop.tsx

```diff
@@ -0,0 +1,155 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+
+import SpeechDesktop from "./assets/speech-desktop.svg";
+
+import ListItemButton from "./components/ListItemButton";
+import TriangleIcon from "./components/TriangleIcon";
+
+interface FAQAccordion {
+	faq: {
+		_key: string;
+		question: JSX.Element;
+		answer: JSX.Element;
+	}[];
+}
+
+const FAQAccordionDesktop: React.FC<FAQAccordion> = ({ faq }) => {
+	const faqGroup1 = faq.slice(0, 8);
+	const faqGroup2 = faq.slice(8);
+
+	const [page1Selected, setPage1Selected] = useState<boolean>(true);
+
+	const [focusedQuestion, setFocusedQuestion] = useState<null | {
+		_key: string;
+		question: JSX.Element;
+		answer: JSX.Element;
+	}>(null);
```

Comment:
Suggestion: rather than redefining the structure of the "faqs," create another interface for each individual "faq" and refactor the `FAQAccordion` interface accordingly